### DISCUSSION TITLE
Integrate BotPress With ChatWoot

### DISCUSSION THREAD

**Discussion by voko3456** [chatwoot/chatwoot#3301] [Feature Requests]
BotPress ChatBot is open source chatbot support building advanced bot with flow builder   
https://github.com/botpress/botpress

**Comment by sojan-official** [comment]
ref: https://github.com/chatwoot/chatwoot/issues/2811

**Comment by douglara** [comment]
Follow this bridge to botpress:
https://github.com/douglara/chatwoot-botpress-bridge
